releases/release-1.33/release-notes/release-notes-draft.md

```diff
@@ -3,7 +3,7 @@
 ### (No, really, you MUST read this before you upgrade)
 
 - Added the ability to reduce both the initial delay and the maximum delay accrued between container restarts for a node for containers in `CrashLoopBackOff` across the cluster to the recommended values of `1s` initial delay and `60s` maximum delay. To set this for a node, turn on the feature gate `ReduceDefaultCrashLoopBackOffDecay`. If you are also using the feature gate `KubeletCrashLoopBackOffMax` with a configured per-node `CrashLoopBackOff.MaxContainerRestartPeriod`, the effective kubelet configuration will follow the conflict resolution policy described further in the documentation [here](TODO:link). ([#130711](https://github.com/kubernetes/kubernetes/pull/130711), [@lauralorenz](https://github.com/lauralorenz)) [SIG Node and Testing]
```

Comment:
We should address this TODO and add the link to the docs.

Reply:
Yeah, we have to add the doc which was merged as part of https://github.com/kubernetes/website/pull/50065
adding this https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#reduced-container-restart-delay
It will not be visible right now but after the release, and website updates, it should.